docker/config/docker_common.env

```diff
@@ -77,8 +77,6 @@ resource.boto.resource_class=socorro.external.boto.connection_context.HostPortS3
 # processor
 # ---------
 
-minidump_stackwalk_pathname=/stackwalk/stackwalker
```

Comment:
`BreakpadStackwalkerRule` uses this, but `BreakpadStackwalkerRule2015` does not. When I added this, I was confused by which rule we were using. So we can nix this line.